view/adminhtml/layout/sales_order_view.xml

```diff
@@ -23,5 +23,8 @@
                    name="bolt.transaction.target"
                    template="Bolt_Boltpay::boltpay/js/transaction.phtml"/>
         </referenceContainer>
+        <referenceContainer name="payment_additional_info">
+            <block class="Bolt\Boltpay\Block\Adminhtml\Order\View\Tab\Info" name="payment_additional_info.cc_info" template="Bolt_Boltpay::order/view/tab/info/payment.phtml"/>
```

Comment:
Is it unique name?
`name="payment_additional_info.cc_info"`
To prevent matches with some other blocks, i would rather to have it like `name="bolt.payment_additional_info.cc_info"`

Reply:
@YevhenShBolt I updated the code